### ISSUE TITLE
Make the turnkey solution rancher specific

### ISSUE THREAD

**Issue by mak3r** [mak3r/turnkey#12]
- Change the hostname to `rancher`
- Change the user to `k3s`
- Change the pw to `rancher`

Review all containers and setup/configuration code to see if there are any assumptions about user/pw/hostname which also need to change.
* resolv.conf
* ui image
* kubeconfig location

**Comment by bgulla** [comment]
In the instance of deploying multiple turnkey devices, we may want to allow the user to set a device-specific hostname. 

I agree though, the default hostname, username, password should be more flavored to the rancher suite.

**Comment by mak3r** [comment]
Yes. A device specific hostname should be fairly straightforward. One would only need to collect any configuration specific info in the UI during the hosted ap process and then how that is used and managed would be done by other processes. Similar to how picking up the SSID and passphrase is done in the MVP
